test/SafeAnchorSpec.js

```diff
@@ -39,6 +39,17 @@ describe('SafeAnchor', () => {
     handleClick.should.have.been.calledOnce;
   });
 
+  it('provides onClick handler as onKeyDown handler for "space"', () => {
+    const handleClick = sinon.spy();
+
+    tsp(<SafeAnchor onClick={handleClick} />)
+      .shallowRender()
+      .find('a')
+      .trigger('keyDown', { key: ' ', preventDefault() {} });
+
+    handleClick.should.have.been.calledOnce;
```

Comment:
yeah this is fine as long as we're not using assert – i prefer `expect` but consistency has some value